cylc/flow/unicode_rules.py

```diff
@@ -351,6 +351,8 @@ class TaskOutputValidator(UnicodeRuleChecker):
         not_starts_with('_cylc'),
         # blacklist keywords
         not_equals('required', 'optional', 'all', 'and', 'or'),
+        # blacklist Run Modes:
+        not_equals(RunMode.SKIP),
```

Comment:
Cylc doesn't generate any "skip" output? What is this trying to achieve?

Reply:
Users can define outputs!